plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreDatabase.java

```diff
@@ -1350,6 +1356,19 @@ protected PostgreSchema fetchObject(@NotNull JDBCSession session, @NotNull Postg
             }
             return owner.createSchemaImpl(owner, name, resultSet);
         }
+
+        /**
+         *
+         * Adds condition in the query and returns true if condition is added.
+         *
+         * @param session to check columns existing
+         * @param query query text needed for additions
+         * @return true if condition added
+         */
+        public boolean addExtraCondition(@NotNull JDBCSession session, @NotNull StringBuilder query) {
```

Comment:
Doesn't make sense to expose it to the world outside
```suggestion
        protected boolean addExtraCondition(@NotNull JDBCSession session, @NotNull StringBuilder query) {
```